flink-sliding-feature-view-benchmark/main.py

```diff
@@ -0,0 +1,156 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import argparse
+import time
+from datetime import timedelta
+
+from feathub.common import types
+from feathub.common.types import Int64
+from feathub.feathub_client import FeathubClient
+from feathub.feature_tables.sinks.black_hole_sink import BlackHoleSink
+from feathub.feature_tables.sources.datagen_source import DataGenSource, RandomField
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.sliding_feature_view import SlidingFeatureView
+from feathub.feature_views.transforms.sliding_window_transform import (
+    SlidingWindowTransform,
+)
+from feathub.table.schema import Schema
+
+
+def run(records_num: int):
+    client = FeathubClient(
+        props={
+            "processor": {
+                "type": "flink",
+                "flink": {"rest.address": "localhost", "rest.port": 8081},
+            },
+            "online_store": {
+                "types": ["memory"],
+                "memory": {},
+            },
+            "registry": {
+                "type": "local",
+                "local": {
+                    "namespace": "default",
+                },
+            },
+            "feature_service": {
+                "type": "local",
+                "local": {},
+            },
+        }
+    )
+    purchase_events_schema = (
+        Schema.new_builder()
+        .column("user_id", types.Int32)
+        .column("item_id", types.Int32)
+        .column("item_count", types.Int32)
+        .column("timestamp", types.Timestamp)
+        .build()
+    )
+    purchase_events_source = DataGenSource(
+        name="purchase_events",
+        schema=purchase_events_schema,
+        number_of_rows=records_num,
+        rows_per_second=records_num,
```

Comment:
Would it be more readable to set this to MAX_LONG, to indicate that the source should send `records_num` records as fast as possible?